sympy/functions/special/error_functions.py

```diff
@@ -556,6 +556,7 @@ def _eval_rewrite_as_uppergamma(self, z, **kwargs):
         from sympy import uppergamma
         return sqrt(-z**2)/z*(uppergamma(S.Half, -z**2)/sqrt(S.Pi) - S.One)
 
+
```

Comment:
It is recommended ([PEP 8](https://www.python.org/dev/peps/pep-0008/#id21)) that method definitions inside a class be separated by a single blank line. Two blank lines are used between different classes.